### ISSUE TITLE
darknet2caffe

### ISSUE THREAD

**Issue by charmowen** [OAID/Tengine#51]
请问您提供的yolo.caffemodel模型是darknet模型转过来的吗？转换的方法链接有吗？谢谢

**Comment by lyuchuny3** [comment]
Yes, the yolo.caffemodel in `Tengine-model-zoo` is converted from the darknet model to caffemodel. You can refer to this: [https://github.com/gklz1982/caffe-yolov2](url).
